pkg/engine/jmespath/functions.go

```diff
@@ -307,6 +308,17 @@ func GetFunctions(configuration config.Configuration) []FunctionEntry {
 		ReturnType: []jpType{jpAny},
 		Note:       "divisor must be non-zero, arguments must be integers",
 	}, {
+		FunctionEntry: gojmespath.FunctionEntry{
+			Name: round,
+			Arguments: []argSpec{
+				{Types: []jpType{jpAny}},
+				{Types: []jpType{jpAny}},
```

Comment:
Shouldn't the second argument always be an int ?

Reply:
Yeah, thanks for pointing it out. I will update the PR.